### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA announces additional inappropriate information when "Settings" dialog   is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1483]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. Settings dialog will open.
4. Observe the Jaws and NVDA behaviour 

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Settings" dialog is opened 

**Jaws behaviour:**
 Settings dialog
Show: Sort by: name Android SupportVersion: 10.2.3 Check or uncheck a plugin to enable or disable it. 
Tree view
Plugins
4 of 10
To move through or expand items use the Arrow keys.

**NVDA Behaviour:**
Settings  dialog  Show:
Sort by: name

Android Support
Version: 10.2.3
Check or uncheck a plugin to enable or disable it.
tree view
Plugins  selected  4 of 8  level 2

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e  Settings dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![settings -opened](https://user-images.githubusercontent.com/22956130/36105964-f7550e88-103b-11e8-9aac-da5e82d1a22a.jpg)
![settings-nvda](https://user-images.githubusercontent.com/22956130/36105977-02f2247e-103c-11e8-8728-3768bdbec505.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749961
